.eslintrc

```diff
@@ -52,7 +52,7 @@
     "no-undef-init": "error",
     "no-nested-ternary": "error",
     "no-unneeded-ternary": "error",
-    "camelcase": "error",
```

Comment:
maybe it's better to add an exception in a particular file `/* eslint-disable camelcase */` ?

Reply:
we already had fe files,  I bel not important rule